File: 04.SERVIDOR/WS_BanQuito_RESTJAVA_GR05/src/main/java/ec/edu/yaranga/BanQuito/dto/CreditoDTO.java
package ec.edu.yaranga.BanQuito.dto;

import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDate;
import java.util.List;
@Data
public class CreditoDTO {
    private Integer idCredito;
    private String cedula;
    private String nombreCliente;
    private BigDecimal monto;
    private Integer plazoMeses;
    private BigDecimal valorCuotaFija;
    private BigDecimal tasaAnual;
    private LocalDate fechaOtorgamiento;
    private String estado;
    private List<AmortizacionDTO> amortizaciones;
}

File: 04. CLI MOV/CLIMOV_ComerElect_RESTJAVA_GR05/app/src/main/java/ec/edu/monster/modelo/banquito/CreditoDTO.kt
package ec.edu.monster.modelo.banquito

import java.math.BigDecimal
import java.time.LocalDate

data class CreditoDTO(
    val idCredito: Int,
    val cedula: String,
    val nombreCliente: String,
    val monto: BigDecimal,
    val plazoMeses: Int,
    val valorCuotaFija: BigDecimal,
    val tasaAnual: BigDecimal,
    val fechaOtorgamiento: LocalDate,
    val estado: String,
    val amortizaciones: List<AmortizacionDTO>
)
